internal/fs/fs.go

```diff
@@ -269,12 +270,16 @@ func CopyDir(src, dst string) error {
 // of the source file. The file mode will be copied from the source and
 // the copied data is synced/flushed to stable storage.
 func copyFile(src, dst string) (err error) {
-	if sym, err := IsSymlink(src); err != nil {
-		return err
-	} else if sym {
-		err := copySymlink(src, dst)
+	sym, err := IsSymlink(src)
+	if err != nil {
 		return err
 	}
+	// Creating symlinks on Windows require an additional permission regular
+	// users aren't granted usually. So we skip cloning the symlink nd copy the
+	// file content as a fall back instead.
+	if sym && runtime.GOOS != "windows" {
```

Comment:
Should work. 

I'm only worried about not having a clear path of execution. The more conditionals paths we have the harder it's to reproduce output or debug strange behavior because the input list grows exponentially.